demisto_sdk/commands/content_graph/strict_objects/base_strict_model.py

```diff
@@ -99,11 +100,25 @@ class ScriptType(StrEnum):
 class StructureError(BaseStrictModel):
     """Used for wrapping Pydantic errors, not part of content."""
 
+    path: Path
     field_name: Optional[tuple] = Field(None, alias="loc")
     error_message: Optional[str] = Field(None, alias="msg")
     error_type: Optional[str] = Field(None, alias="type")
```

Comment:
are they really `Optional`? I think we can rely on Pydantic to always return them.